ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewHeaderFooterViewDemoController.swift

```diff
@@ -57,7 +57,7 @@ class TableViewHeaderFooterViewDemoController: DemoController {
         tableView.register(TableViewHeaderFooterView.self, forHeaderFooterViewReuseIdentifier: TableViewHeaderFooterView.identifier)
         tableView.dataSource = self
         tableView.delegate = self
-        tableView.backgroundColor = TableViewCell.tableBackgroundColor
+        tableView.backgroundColor = UIColor(dynamicColor: view.fluentTheme.aliasTokens.colors[.background2])
```

Comment:
Is the table view background supposed to be background 2 now? If so, I think it'd be better to change that inside of TableViewCell instead of doing it here.

Reply:
Nah, it's correct, I just wanted the demo to look consistent!